ui-core/src/main/java/com/airwallex/android/ui/AirwallexActivityLaunch.kt

```diff
@@ -4,14 +4,62 @@ import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
 import android.os.Parcelable
+import androidx.activity.ComponentActivity
+import androidx.activity.result.ActivityResult
+import androidx.activity.result.ActivityResultCallback
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleObserver
+import androidx.lifecycle.LifecycleOwner
 
 abstract class AirwallexActivityLaunch<TargetActivity : Activity, ArgsType : AirwallexActivityLaunch.Args> constructor(
     private val originalActivity: Activity,
     private val fragment: Fragment? = null,
     private val targetActivity: Class<TargetActivity>,
     private val requestCode: Int
 ) {
+    companion object {
+        private var resultLauncher: ActivityResultLauncher<Intent>? = null
+        private var launchObserver: LifecycleObserver? = null
+        private var launchRequestCode = 0
```

Comment:
`handlePaymentData` in `AirwallexStarter` can't actually handle this request code 0 right now, are you sure it's working?

Reply:
Oh I see, this 0 is a placeholder, maybe we should put it as `Int?` without default value to avoid confusion.

Reply:
OK，I put default value as null